<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QLineEdit" name="trProtInput"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="frInput"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="ReadProtInput"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="freqLabel">
      <property name="text">
       <string>Frequency</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="pathLabel">
      <property name="text">
       <string>PATH</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="pathInput"/>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="halPort">
      <property name="text">
       <string>PORT</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="start">
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLineEdit" name="fgPortInput"/>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="halPortInput"/>
    </item>
    <item row="9" column="0">
     <widget class="QLineEdit" name="fgIpInput"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="fgPort">
      <property name="text">
       <string>PORT</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="fgIp">
      <property name="text">
       <string>IP address</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="halIp">
      <property name="text">
       <string>IP address</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="halIpInput"/>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="halConsole">
      <property name="text">
       <string>Console</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QTextBrowser" name="fgConsoleOutput">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="fgConsole">
      <property name="text">
       <string>Console</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QTextBrowser" name="halConsoleOutput"/>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>--&lt;HAL&gt;--</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>--&lt;FlightGear&gt;--</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Transmit protocol (ex: flierin)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Read protocol (ex: flierout)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
